packages/accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,60 @@
+import { PropsWithChildren, ReactNode } from 'react';
+import { Box, BoxProps } from '@ag.ds-next/box';
+import { useToggleState } from '@ag.ds-next/core';
+import { useId } from '@reach/auto-id';
+
+import { AccordionTitle, AccordionTitleProps } from './AccordionTitle';
+import { AccordionBody, AccordionBodyProps } from './AccordionBody';
+
+type AccordionProps = {
+	id?: string;
+	isInitiallyOpen?: boolean;
+} & Pick<AccordionTitleProps, 'titleHeadingLevel' | 'title'> &
+	Pick<AccordionBodyProps, 'noPadding'> &
+	Pick<BoxProps, 'palette'>;
+
+export const useAccordionIds = (initialId: string | undefined) => {
+	const id = useId(initialId);
+	return {
+		titleId: `${id}-title`,
+		bodyID: `${id}-default`,
+	};
+};
+
+export const AccordionItem = ({
+	isInitiallyOpen = false,
+	noPadding,
+	children,
+	palette,
+	title,
+	titleHeadingLevel,
+	...props
+}: PropsWithChildren<AccordionProps>) => {
+	const { titleId, bodyID } = useAccordionIds(props.id);
+	const [isOpen, onToggle] = useToggleState(isInitiallyOpen, !isInitiallyOpen);
```

Comment:
What's this second parameter to `useToggleState` 🤔 Is it required?

Reply:
It is `alternateValue`, so the state doesn't necessarily have to be a boolean